test-utils/src/main/groovy/io/micronaut/starter/test/CommandSpec.groovy

```diff
@@ -60,14 +60,15 @@ abstract class CommandSpec extends Specification {
 
     String executeMaven(String command) {
         if (OperatingSystem.current.isWindows()) {
-            command = "mvnw.bat " + command
+            command = dir.getAbsolutePath()+"\\"+"mvnw.bat " + command
```

Comment:
Is this supposed to be working on Unix?

Reply:
Yes, I only updated the section where OperatingSystem.current.isWindows.  I made no modifications to anything linux/Unix based.

Reply:
Oh, I'm sorry, missed the part about being Windows-specific.